src/components/Avatar/Avatar.vue

```diff
@@ -421,6 +421,12 @@ export default {
 		background-color: var(--color-text-maxcontrast);
 	}
 
+	&:not(&--unknown) {
+		// White background for avatars with transparency, also in dark theme
+		background-color: #fff !important;
+		box-shadow: 0 0 5px rgba(0, 0, 0, 0.05) inset;
```

Comment:
For the antialiasing?

Reply:
@skjnldsv what do you mean by that question?

Reply:
I asked the reasons of this line? :)

Reply:
When you touch css, I would love to have more comments on why you're adding this or this, I recently reviewed some of your css and sometimes I don't understand exactly.

Please abuse css comments, it gets removed on compile anyway  ;)

Reply:
Check the screenshots, this is so that there is a circle around it. :) Could also be a border, but then it would show for all images which doesn’t look good. This will only show for images with transparency.

Reply:
I mean code comment, not github ;)